### ISSUE TITLE
Initial submission to eLife: timeline & updates

### ISSUE THREAD

**Issue by agitter** [greenelab/meta-review#8]
We are fairly close to having a full draft ready for submission.  @dhimmel and @cgreene do you have any thoughts on a timeline?  The remaining tasks are the Manubot section, addressing the many inline TODOs, and perhaps a figure to illustrate the issue and pull request system.

**Comment by dhimmel** [comment]
@agitter I'll get to work! Hopefully I can knock most of the Manubot content out next week... I begun yesterday!

**Comment by agitter** [comment]
Great!  I think I nagged @cgreene enough to get it on his TODO list also.

Any thoughts on a figure?  I have no graphic design skill so if I draft it, it would have to be screenshot-based.

**Comment by cgreene** [comment]
I have addressed the major TODOs that I think were assigned to me. :smile: I agree that a quick timeline is good.

**Comment by agitter** [comment]
What do both of you think about a figure?  If we skip that, we could finish a full draft within a week.

**Comment by dhimmel** [comment]
> What do both of you think about a figure? If we skip that, we could finish a full draft within a week.

I think we should have a figure or two. Here are areas of interest:

1. screenshots of the contribution life cycle. Pull request / review comments > merged commit > Travis CI build / updated manuscript.
2. citation exposé showing how the Manubot does citations / references
3. Deep Review contribution plot. Here are possibilities: [Rephetio](http://git.dhimmel.com/rephetio-manuscript/#fig:contribution), [Cognoma](https://github.com/cognoma/cognoma#contributing), or a Gource-movie [like this](https://www.youtube.com/watch?v=dTILX-_JzTs).

**Comment by agitter** [comment]
Those are all excellent ideas.  1 is the most essential because it will help readers understand the main idea and contribution.

For 3, are those figures easy to generate automatically from a commit history or does it take some manual effort?  What would the cognoma graphic look like for us because we only have one repo?  Markdown files by users?  I like the static figures better than the video.

**Comment by agitter** [comment]
I'm following up on this figure idea
> screenshots of the contribution life cycle. Pull request / review comments > merged commit > Travis CI build / updated manuscript.

I looked through some of the deep review pull requests.  greenelab/deep-review#286 and greenelab/deep-review#347 stood out as two where there was discussion, user tagging, issue referencing that will illustrate advantages of GitHub.  We could show panels with:

1. Review comments
![image](https://user-images.githubusercontent.com/3532898/32682429-363ceaea-c63a-11e7-82f4-19b817570ccc.png)
2. Updates in response to review
![image](https://user-images.githubusercontent.com/3532898/32682433-47eeb76e-c63a-11e7-95ef-b68c814c7e03.png)
3. The final changes from the pull request
![image](https://user-images.githubusercontent.com/3532898/32682467-6c3e4d82-c63a-11e7-9773-e11a894d2db5.png)

@cgreene @dhimmel Any other ideas for what we want to show in this figure?  If the changes were smaller in scope, it would be easier to show the edits to the full manuscript.  The third image is probably too big to include.  But the smaller pull requests didn't seem to generate as much review discussion of course.

**Comment by cgreene** [comment]
Hmm, I think I might just do part 1 and 2. It's nice to see the changes overall but the "before" part of that screenshot isn't too interesting. I agree though, it's a lot of text. I wonder if a schematic in the paper and the full example as a supplementary figure is better...

**Comment by agitter** [comment]
You're right, there isn't much interesting content per pixel there.  I agree that a schematic would be much better than screenshots.  I just don't have the graphical skills to make a nice one.

If none of us can make a schematic, should we create a new pull request designed to showcase these features instead of trying to find an existing one?  It doesn't have to be from deep review, we could engineer a small change here to fix a TODO or add a reference.

**Comment by cgreene** [comment]
![manubot-draft](https://user-images.githubusercontent.com/542643/34302462-f917d07c-e6fe-11e7-9b5f-691ac7c21bf4.png)

I tried just to keep the focus on the most important bits. I made it in powerpoint, so I have a pptx file.

**Comment by cgreene** [comment]
here's the file in case you want to edit https://www.dropbox.com/s/7nqn77igjophuhz/manubot-draft.pptx?dl=0

**Comment by agitter** [comment]
This simplified version is _much_ better than a raw screenshot.  Want to make a pull request to add this figure?  You can also put the pptx in `content/images` in case we need to edit it later.

I had de-prioritized this manuscript when I got caught up with others, but do we want to try to have a finished draft before we resubmit deep review?  It would help simplify our deep review methods section.

**Comment by dhimmel** [comment]
> do we want to try to have a finished draft before we resubmit deep review?

I actually think the Meta Review will be best after the deep review resubmission, when the contribution figure stabilizes. Both the Manubot and Deep Review are still progressing rapidly, so the Meta Review will get incomplete / outdated quickly.

> It would help simplify our deep review methods section.

I'll take another look at the Deep Review methods.

**Comment by agitter** [comment]
> when the contribution figure stabilizes

That makes sense now that we have an ongoing burst of deep review contributions.

**Comment by agitter** [comment]
Now that deep review has been resubmitted, I wanted to ping both of you (@cgreene @dhimmel) to think about our timing again.  This is not really a good time for me to work on this, and I see you are both working on SciHub (among other things).  However, the anticipated deep review coverage might provide a golden opportunity to submit meta review while interest in the process is high.

What do you think?

**Comment by cgreene** [comment]
I think that we will never have a "good" time to work on something like this, but this is probably the right time to work on it. We should press onward and get it submitted. I can shift some focus to this on or about Feb 5 if that's ok. I've got a trip next week that should be more or less all consuming.

**Comment by agitter** [comment]
Sounds good to me.  I'll block off some time for this starting Feb 3 or 4.

@dhimmel I won't edit anything until we merge #26.

**Comment by dhimmel** [comment]
>  I won't edit anything until we merge #26.

Okay I'll try to resume work on #26 next week.

> the anticipated deep review coverage might provide a golden opportunity to submit meta review while interest in the process is high

I agree. I think finishing v1 of the meta review shortly after the deep review is published makes sense.

**Comment by agitter** [comment]
@vsmalladi @slochower we should be ready to submit in the next few days.  We initially plan to submit this to _eLife_ as a feature article.

**Comment by vsmalladi** [comment]
@agitter Is there a final review that we should be looking out for that we will be approving?

**Comment by dhimmel** [comment]
> Is there a final review that we should be looking out for that we will be approving?

The current version is the planned submission version, as far as I know. So if you have any further comments on https://greenelab.github.io/meta-review/v/b8eeea542ce238bbcaf2023add2aecb86ef726bd/, let us know!

**Comment by slochower** [comment]
I just added a [PR](https://github.com/greenelab/meta-review/pull/94) with one grammar correction, one spelling change, and one sentence rearrangement.

I also made a comment on https://github.com/greenelab/meta-review/commit/17b42d887906c1ba81ce81fcc64e4a4319dc384d about a phrase I didn't understand.

I think it reads quite nicely. I started to get a sense that the introduction stressed *review* articles too heavily, but since the discussion mentions all the other types of Manubot documents out there, I think this forms a powerful sandwich. I think this is good to go from my point of view.
